src/logic/FetchRemotes.ts

```diff
@@ -51,17 +83,20 @@ export async function getTaggedRepos(tag: RepoTopic, page = 1, BLACKLIST:string[
 * @returns The manifest object
 */
 async function getRepoManifest(user: string, repo: string, branch: string) {
-  const sessionStorageItem = window.sessionStorage.getItem(`${user}-${repo}`);
-  const failedSessionStorageItems = window.sessionStorage.getItem("noManifests");
+  const key = `${user}-${repo}`;
```

Comment:
Do we not need the branch anymore? What if the user has one repo and uses different branches for different projects?

Reply:
That's still the same key we used before, just initialized once.
Also one repo only has one manifest. This caches everything in the manifest file, not one manifest object itself.